datascience/tables.py

```diff
@@ -1328,98 +1328,110 @@ def with_rows(self, rows):
         return self
 
     def with_column(self, label, values):
-        """Return a table with an additional or replaced column.
+        """Return a new table with an additional or replaced column.
 
         Args:
             ``label`` (str): The column label. If an existing label is used,
-                that column will be replaced in the returned table.
+                the existing column will be replaced in the new table.
 
             ``values`` (single value or sequence): If a single value, every
-                value in the new column is ``values``.
+                value in the new column is ``values``. If sequence of values, 
+                new column takes on values in ``values``. 
 
                 If a sequence, the new column contains the values in
-                ``values``. ``values`` must be the same length as the table.
+                ``values``, must be the same length as the table.
```

Comment:
Isn't this kind of a repeat of the last line but with the length constraint? Feels like you should just mention the length constraint here.